src/createServer.js

```diff
@@ -16,39 +16,17 @@ function createServer() {
         'Content-Type': 'text/plain',
       });
 
-      res.end(
-        JSON.stringify({
-          error: 'Path to file should be like this /file/folderName/filename',
-        }),
-      );
+      res.end('Path to file should be like this /file/folderName/filename');
 
       return;
     }
 
-    if (!pathName.startsWith('/file')) {
-      res.writeHead(400, {
+    if (!pathName.startsWith('/file/')) {
+      res.writeHead(200, {
         'Content-Type': 'text/plain',
       });
 
-      res.end(
-        JSON.stringify({
-          error: 'Path to file should be like this /file/folderName/filename',
-        }),
-      );
-
-      return;
-    }
-
-    if (arrayFromPathName.includes('..')) {
-      res.writeHead(400, {
-        'Content-Type': 'text/plain',
-      });
-
-      res.end(
-        JSON.stringify({
-          error: 'Path to file should be like this /file/folderName/filename',
-        }),
-      );
```

Comment:
Only 1 failed test remains and to fix it you need to change this condition instead of deleting it entirely
![image](https://github.com/user-attachments/assets/7605f248-36ea-451c-8a8c-d1018aa3a9ce)